js/gated-content/src/components/event/EventListing.vue

```diff
@@ -208,10 +235,7 @@ export default {
 
       params.filter.status = 1;
       params.sort = {
-        sortByDate: {
-          path: 'date.value',
-          direction: 'ASC',
-        },
+        sortByDate: this.sort,
```

Comment:
Let's move to `sortBy` for the consistency

Reply:
Fixed